.github/workflows/release.yml

```diff
@@ -9,7 +9,6 @@ on:
   push:
     branches: [ 'master', 'main', 'release-*' ]
     # Publish semver tags as releases.
-  pull_request: {}
```

Comment:
maybe these lines can be removed as well?
https://github.com/kubewharf/katalyst-core/pull/3/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R78-R82
and we can change this line to `push: true`
https://github.com/kubewharf/katalyst-core/pull/3/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R93

Reply:
Yes, I think these lines are no longer necessary, and I have modified them.